rancher/rancher-compose/rancher-services/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Custom Rancher Services
+layout: rancher-default
+
+---
+
+## Custom Rancher Services
+
+Custom Rancher services are configured by using a special image name in the compose template.  The image name is how `rancher-compose` knows to set up a Rancher service versus a normal service.
+
+Service | Image Name
+--------|-----------
+Load Balancer | rancher/load-balancer-service
+External Service | rancher/external-service
+Alias/DNS Service | rancher/dns-service
+
+### Load Balancer
+
+A load balancer can be scheduled like any other service. Read more about [scheduling]({{site.baseurl}}/rancher/rancher-compose/scheduling/) services and load balancers using `rancher-compose`.
+
+Load balancing using the basic use case of load balancers is simply adding ports and linking services. Any traffic directed to a source port will be sent to the private port of the linked services.
+
+We go into more detail on [advanced routing options]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-balancers/#advanced-routing-optons) for load balancers in our UI section, but anything that we can create in Rancher can be created using `rancher-compose`.
+
+Advanced routing options use `labels` in the `docker-compose.yml` file. Here's the basic syntax for applying the label. You would only use this label if you were going to do some advanced routing rules.
+
+Label Key | Label Value
+---| ---
+`io.rancher.loadbalancer.target.SERVICE_NAME` | `REQUEST_HOST:SOURCE_PORT/REQUEST_PATH=TARGET_PORT`
+
+> **Note:** If you use a source port with the advanced routing options, the source port must also be listed in `ports` section of the `docker-compose.yml`
+
+Here is the syntax for all combinations using request host, source port, path and target port. 
+
+Request Host | Source Port | Path | Target Port | Label Value
+---|---|---|---| ---
+example.com | 80 | path | 81 | `example.com:80/path=81`
+example.com | 80 | path | | `example.com:80/path`
+example.com | 80||81 | `example.com:80=81`
+example.com |80 | | | `example.com:80`
+example.com | | path | 81 | `example.com/path=81`
```

Comment:
Would be nice in one of the examples to show that we support multiple "/" in URL path, something like:

example.com:80/foo/bar=81